### ISSUE TITLE
Info / Article: "How Swipe Typing works"

### ISSUE THREAD

**Issue by DocSniper** [florisboard/florisboard#1776]
I found several issues that refer to "fleksy", but none seemed to fit, so I made a separate entry:
Just an info, but maybe this article can help to improve the project:

**How Swipe Typing works**
[https://www.fleksy.com/blog/how-swipe-typing-works/](https://www.fleksy.com/blog/how-swipe-typing-works/)

**Comment by lm41** [comment]
This is a dead link. Closing thus.

**Comment by DocSniper** [comment]
Hi @lm41,

I'm not asking to reopen the issue, I just wanted to add a short clarification.

The link was working when I opened this in 2022, so this seems to be link rot. I also checked archive services, and the article is still available there.

From my perspective, it would have been worth checking that before closing the issue.

I've since moved on to another keyboard, so no further action is needed on my side. I just wanted to leave this as constructive feedback.

Best regards

**Comment by lm41** [comment]
Yeah I figured that the link was working in 2022 but since then a long time has passed and the link is no longer working. I always forget that the internet archive exists 🤦‍♂️. I've updated your original post to include the archive link.
